crates/oxc_prettier/src/format/mod.rs

```diff
@@ -1340,10 +1340,6 @@ impl<'a> Format<'a> for NumericLiteral<'a> {
             let raw = self.span.source_text(p.source_text);
             let mut string = Cow::Borrowed(raw);
 
-            if string.contains(|c: char| c.is_ascii_uppercase()) {
-                string = Cow::Owned(string.to_ascii_lowercase());
```

Comment:
oh ... this should be changed to `string = string.to_ascii_lowercase()`?

Reply:
Yes. I misunderstood "remove contains check".